chore: update CHANGELOG to better convey the 2.0.0 issue (#9007)
ZcashFoundation/zebra: CHANGELOG.md

## [Zebra 2.0.1](https://github.com/ZcashFoundation/zebra/releases/tag/v2.0.1) - 2024-10-30

- Zebra now supports NU6 on Mainnet. This patch release updates dependencies
  required for NU6. The 2.0.0 release was pointing to the incorrect dependencies
  and would panic on NU6 activation.

### Breaking Changes

- The JSON RPC endpoint has cookie-based authentication enabled by default.

>   **If you rely on Zebra RPC, wou will need to adjust your config.** The

Reviewer: ```suggestion
  **If you rely on Zebra RPC, you will need to adjust your config.** The
```

  simplest change is to disable authentication by adding `enable_cookie_auth =
  false` to the `[rpc]` section of the Zebra config file; [refer to the

>   docs](https://zebra.zfnd.org/user/lightwalletd.html#json-rpc). (This was added
>   in 2.0.0 but we include here again for clarity)

Reviewer: ```suggestion
  docs for more information](https://zebra.zfnd.org/user/lightwalletd.html#json-rpc) (this was added
  in v2.0.0, but is being mentioned again here for clarity).
```

### Changed

- Use ECC deps with activation height for NU6
  ([#8960](https://github.com/ZcashFoundation/zebra/pull/8978))

### Contributors

Thank you to everyone who contributed to this release, we couldn't make Zebra without you:
@arya2, @gustavovalverde, @oxarbitrage and @upbqdn.

## [Zebra 2.0.0](https://github.com/ZcashFoundation/zebra/releases/tag/v2.0.0) - 2024-10-25 - [YANKED]

This release was intended to support NU6 but was pointing to the wrong version

> of dependencies which would make Zebra panic at NU6 activation. Use 2.0.1 instead.

Reviewer: ```suggestion
of dependencies which would make Zebra panic at NU6 activation. Use v2.0.1 instead.
```

### Breaking Changes

- Zebra now supports NU6 on Mainnet.
- The JSON RPC endpoint has a cookie-based authentication enabled by default.

>   **If you rely on Zebra RPC, wou will need to adjust your config.** The

Reviewer: ```suggestion
  **If you rely on Zebra RPC, you will need to adjust your config.** The
```
